package.json

```diff
@@ -1,6 +1,5 @@
 {
 	"private": true,
-	"type": "module",
```

Comment:
Not sure if this is the right way to make the script run: one solution is to convert both scripts to JS instead. If `type: module` is mentioned, I get the following error:

```
TypeError: Unknown file extension ".ts" for .../refined-github/build/verify-features.ts
```

Let me know if there is a better way to do this. Thanks

Reply:
Seems running `ava` also fails when I remove `type: module`. I'll look for a better way, let me know if you already know one.

Reply:
Yeah this can't be dropped. Probably a solution is in here: https://github.com/TypeStrong/ts-node/issues/1007

I suppose the command is

```sh
node --loader ts-node/esm ./build/verify-features.ts
```

Reply:
Will try that, thanks!